tests/translator/input/http_api_multiple_authorizers.yaml

```diff
@@ -62,9 +62,9 @@ Resources:
             - scope
             IdentitySource: $request.header.Authorization
             JwtConfiguration:
-              audience:
+              Audience:
```

Comment:
Why is the input file changed? It’s hard to gauge what about the transform is being changed, and whether existing templates transform the same.

Reply:
The issue is about removal the confusion created by the fact that `issuer` and `audience` should be lower case when everything else is capitalized. So the point is to change input, run the tests and see that the output has not changed. There is no change in output files in the PR, this is the proof that output stays the same with capitalized `Issuer` and `Audience`

Reply:
Ah okay I see, and just checked we have a bunch of non-changing tests with `audience`, so this is fine.